concrete/src/Authentication/Type/OAuth/BindingService.php

```diff
@@ -93,8 +94,10 @@ public function clearBinding($userId, $binding, $namespace, $matchBoth = false)
                     'binding' => $binding,
                 ]);
 
-                $result = $qb->execute();
-                $total = $result->fetchColumn();
+                $total = $qb->execute();
+                if ($total instanceof Statement) {
```

Comment:
Since the query is of type `QueryBuilder::DELETE`, its `execute` method should execute the `executeUpdate` connection method, which should return an integer...

Reply:
Yeah I'm aware, I just didn't want to break backwards compatibility if this was ever working.